### PR TITLE
fix(news-api): cleanup unused environments

### PR DESCRIPTION
**Pull request by prusnak**
I did some cleanups to the news-api package.

Will test them whether they work as expected and then remove the draft state.

### CODE REVIEW

File: packages/news-api/src/index.ts [hunk 1]
```diff
@@ -48,7 +43,7 @@ const fetcher = async (
     callback: (statusCode: number, data: string | null, errMessage?: string) => void,
 ) => {
     axios
```
Comment (mroz22) [inline packages/news-api/src/index.ts:45]: unrelated. just wondering we use axios here. it is imho completely unnecessary.

Comment (prusnak) [reply packages/news-api/src/index.ts:45]: @vladimirvolek Do you remember why you've chosen axios here? Is it a good idea to drop axios and use `whatwg-fetch`/`node-fetch`?

**Comment by prusnak** [comment]
I tried this locally and it works.

File: packages/news-api/src/index.ts (hunk 1 above)
Comment (vladimirvolek) [reply packages/news-api/src/index.ts:45]: You can drop it if you want and use something else.

Comment (vladimirvolek) [reply packages/news-api/src/index.ts:45]: Btw are you sure the environments are really not used?
https://staging-news.trezor.io/
https://news.trezor.io/

Comment (vladimirvolek) [reply packages/news-api/src/index.ts:45]: https://github.com/trezor/trezor-suite/blob/b010f7a5556be360a6bf9b4a34efebba570cd017/packages/suite/src/hooks/dashboard/useNews.ts#L16

Comment (prusnak) [reply packages/news-api/src/index.ts:45]: Thanks, addressed in 725965616cf469d41322b1c0460b4eb09cb26a3b
Let's use only production news API

**Review by szymonlesisz — APPROVED** [review]

**Pull request merged**